### ISSUE TITLE
Warping Transform Properties as a block?

### ISSUE THREAD

**Issue by hell-oh-world** [renpy/renpy#2641]
So basically you can do this in Ren'Py...
```py
transform sample:
  easein 0.25 xpos 0.5 xanchor 0.0
```
But this becomes troublesome if you have a ton of properties that you want to warp using the same warp statement.
```py
transform sample:
  easein 0.25 prop1 prop1val prop2 prop2val prop3 prop3val
```
Is there any way that we can warp multiple properties as a block instead? Like this...
```py
transform sample:
  easein 0.25: # warp statement as block statement using `:`
    prop1 prop1val prop2 prop2val
    prop3 prop3val
```
In that way, we can group and see clearly what properties are part of that warp statement.
Please let me know if I'm missing something, If this is an existing feature that is done in another way instead.

**Comment by BattleProgrammer-san** [comment]
You can kinda achieve this by stacking transforms.

```
transform generic_sample:
  xpos 0.25
  zoom 1.5
  ypos 0.3
  
transform ease_sample:
  ease 0.25 generic_sample

```

**Comment by renpytom** [comment]
Implemented by cdedb83cd36859b565b1118b5789fb534b2ed7ed .
